### ISSUE TITLE
Start button for light theme fails color contrast test

### ISSUE THREAD

**Issue by ip69719** [ip69719/ci-portfolio-two#7]
Start button for light theme fails color contrast test
<img width="515" alt="image" src="https://user-images.githubusercontent.com/83339812/200826459-9e572b26-ca6d-49ef-85d7-a2d713ae40fe.png">

**Comment by ip69719** [comment]
Replaced values for start button color with CSS variables"

**Comment by ip69719** [comment]
Replaced values for start button color with CSS variables
